contracts/ERC721.sol

```diff
@@ -6,7 +6,7 @@ pragma solidity >= 0.6.2;
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
 
-contract GameItem is ERC721 {
+contract NFT is ERC721 {
```

Comment:
in general, the filename should match the main contract name, so we should rename the file NFT.sol